src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -223,6 +225,7 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 	//Saves scrollTop value on scroll change
 	public scrollHandler(event: Event) {
 		this._scrollTop = (<HTMLElement>event.srcElement).scrollTop;
+		this.model.onScroll.fire();
```

Comment:
I really wish that we had a better mechanism than this and could avoid this anti-pattern that we started with layout change events going through the model. Did the subscriber test not work correctly?

Reply:
Discussed offline. Was not able to utilize the Parent / Child component method to access the Parent events.